### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by waelelsafty07**
## PR Details

- Setup a new React application using [Create React App](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app)
- Install [React Router V6](https://reactrouter.com/en/main/start/tutorial#setup)
- Add 2 routes in your application, each should render a page component
- Contains a list to renders individual books
- Contains a form to create new books
- Contains a navigation to link to the 2 routes you created

### CODE REVIEW

File: src/components/listofbooks/OneBook.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from 'react';
+import Button from '../utils/Buttons';
+
+const OneBook = () => {
+  const [progress, setProgress] = useState(0);
+
+  useEffect(() => {
+    const bar = (65 * 60) / 100;
+    setProgress(bar);
```
Comment (TadesseDev) [inline src/components/listofbooks/OneBook.js]: - [x] Dear @waelelsafty07, for the book component the expected behavior is to take the book data as a prop and render the title and the author. which means, You need to pass the book as a prop form the `ListOfBooks.js` component and use it here. 
something like 
``` Javascript 
listOfBooks=[
// some books here
]
const ListOfBooks = () => (
  <>
{listOfBooks.map(book=> (<OneBook  book={book}/>))}
/*
    <OneBook />
    <OneBook />
    <OneBook />
This is not professional enough to be reused. Imagin rendering a 100 books this way.
*/
  </>
);
```

File: src/components/utils/Author.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const Author = () => (
+  <div className="form-control author">
+    <select id="author" className="form-control-author bg-white " name="author">
+      <option value="wael-elsafty">Wael Elsafty</option>
+      <option value="seif-ali">Seif Ali</option>
+    </select>
+  </div>
+);
```
Comment (TadesseDev) [inline src/components/utils/Author.js]: - [x] Dear @waelelsafty07, Great job on using a `util` for rendering such components, But please notice the `Author` field is an `input` element with a type of `text`. Please change the `select` type to the `input` type. we need to be able to write the author's information.

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi Team

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7




## Highlights
 - [x]  Correct GitFlow  ✅
 - [x]  Good commit history  ✅
 - [x]  Good PR title/description  ✅
 - [x]  All linters passed  ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by binhussen — APPROVED** [review]
Hi @waelelsafty07 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- Fixed Reviewer Seggesions ✔️
- Used  react-router-dom ✔️
- Used Git flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**